src/main/java/org/fineract/messagegateway/sms/providers/impl/wirepick/smsgateway/model/MsgStatus.java

```diff
@@ -0,0 +1,61 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.fineract.messagegateway.sms.providers.impl.wirepick.smsgateway.model;
+
+public class MsgStatus {
+	
+	private String msgid;
+	private String phone;
+	private String status;
+	private int messageId;
+
+	public String getMsgid() {
```

Comment:
It would be better if you can change this to `getMessageId()`, descriptive names are always preferred.

Reply:
i do agree

Reply:
this is not resolved yet